test/automerge-unit.jl

```diff
@@ -29,6 +29,17 @@ const AutoMerge = RegistryCI.AutoMerge
         @test !AutoMerge.meets_name_length("Flux")[1]
         @test !AutoMerge.meets_name_length("Flux")[1]
     end
+    @testset "Package names is ASCII" begin
+        @test !AutoMerge.meets_name_ascii("ábc")[1]
+        @test AutoMerge.meets_name_ascii("abc")[1]
+    end
+    @testset "Package name distance" begin
+        @test AutoMerge.meets_distance_check("Flux", ["Abc", "Def"])[1]
+        @test !AutoMerge.meets_distance_check("Flux", ["FIux", "Abc", "Def"])[1]
+        @test !AutoMerge.meets_distance_check("Websocket", ["Websockets"])[1]
```

Comment:
Might want to compare with `WebSockets` since that's the actual name of the existing package and the change of case affects the edit distance potentially. Are you doing the edit distance counting case changes or not?

Reply:
Whoops, fixed the test, and now am lowercasing for the edit distances.